meta/MetaKeyHasher.cpp

```diff
@@ -160,12 +160,20 @@ static inline std::size_t sai_get_hash(
     return ne.ip_address.addr_family;
 }
 
+static_assert(sizeof(uint32_t) == 4, "uint32_t expected to be 4 bytes");
```

Comment:
>static_assert [](start = 0, length = 13)

This assertion seems not needed. Do you anticipate some failure case?

Reply:
just a surprise for some wired platforms :D